apiserver/facades/client/machinemanager/instance_information.go

```diff
@@ -31,10 +32,20 @@ func instanceTypes(mm *MachineManagerAPI,
 	}
 
 	cloudSpec := func() (environs.CloudSpec, error) {
-		cloudName := model.Cloud()
+		cloud, err := model.Cloud()
```

Comment:
This logic seems to be duplicated in `apiserver/facades/client/machinemanager/instance_information.go`. Would it be worth extracting?